src/main/webapp/pipe.js

```diff
@@ -311,124 +283,150 @@ function pipelineUtils() {
-                    html.push("<td class='pane-header' style='font-size: smaller; vertical-align: bottom;'>Normal</td>");
-                    html.push("<td class='pane-header' style='font-size: smaller; vertical-align: bottom;'>Low</td>");
-                html.push("</tr>");
-            html.push("</thead>");
-            html.push("<tbody>");
+        var html = ['<div class="infoPanelOuter">'];
+        html.push('<div class="infoPanel"><div class="infoPanelInner">');
+            html.push('<table id="priority.summary" class="pane">');
+            html.push('<thead>');
+                html.push('<tr>');
+                    html.push('<td class="pane-header">Warnings</td>');
+                    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">High</td>');
+                    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">Normal</td>');
+                    html.push('<td class="pane-header" style="font-size: smaller; vertical-align: bottom">Low</td>');
+                html.push('</tr>');
+            html.push('</thead>');
+            html.push('<tbody>');
             Q.each(task.staticAnalysisResults, function(i, analysis) {
-                html.push("<tr>");
-                    html.push("<td class='pane'><a href=" + getLink(data,analysis.url) + ">" + trimWarningsFromString(analysis.name) + "</a></td>");
-                    html.push("<td class='pane' style='text-align: center;'>" + analysis.high + "</td>");
-                    html.push("<td class='pane' style='text-align: center;'>" + analysis.normal + "</td>");
-                    html.push("<td class='pane' style='text-align: center;'>" + analysis.low + "</td>");
-                html.push("</tr>");
+                html.push('<tr>');
+                    html.push('<td class="pane"><a href=' + getLink(data,analysis.url) + '>' + trimWarningsFromString(analysis.name) + '</a></td>');
+                    html.push('<td class="pane" style="text-align: center">' + analysis.high + '</td>');
+                    html.push('<td class="pane" style="text-align: center">' + analysis.normal + '</td>');
+                    html.push('<td class="pane" style="text-align: center">' + analysis.low + '</td>');
+                html.push('</tr>');
             });
-            html.push("</tbody>");
-            html.push("</table>");
-        html.push("</div></div>");
-        html.push("</div>");
-        return html.join("");
+            html.push('</tbody>');
+            html.push('</table>');
+        html.push('</div></div>');
+        html.push('</div>');
+        return html.join('');
     }
 }
 
 function trimWarningsFromString(label) {
-    var offset = label.indexOf("Warnings");
-    if (offset == -1) {
-        return label;
-    } else {
-        return label.substring(0, offset).trim()
-    }
+    var offset = label.indexOf('Warnings');
+    return offset == -1
```

Comment:
What is the reasoning behind using the ? : syntax in favor of an if / else here?